.travis.yml

```diff
@@ -21,7 +21,24 @@ before_script:
     fi
 script: "node ./scripts/build.js"
 after_success:
-  - if [ $IS_PUSH_TO_MASTER ]; then
-      node ./scripts/build-docs.js;
+  - if [ $IS_PUSH_TO_MASTER && "${TRAVIS_JOB_NUMBER}" == *1 ]; then
```

Comment:
Ah, is it possible to add an inline comment of some sort there? Even just a link to this discussion if nothing better is out there.